dev/integration_tests/android_views/lib/wm_integrations.dart

```diff
@@ -0,0 +1,104 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+
+import 'page.dart';
+
+class WindowManagerIntegrationsPage extends PageWidget {
+  const WindowManagerIntegrationsPage()
+      : super('Window Manager Integrations Tests', const ValueKey<String>('WmIntegrationsListTile'));
+
+  @override
+  Widget build(BuildContext context) => WindowManagerBody();
+
+}
+
+class WindowManagerBody extends StatefulWidget {
+  @override
+  State<WindowManagerBody> createState() => WindowManagerBodyState();
+}
+
+enum _LastTestStatus {
+  pending,
+  success,
+  error
+}
+
+class WindowManagerBodyState extends State<WindowManagerBody> {
+
+  MethodChannel viewChannel;
+  _LastTestStatus lastTestStatus = _LastTestStatus.pending;
+  String lastError;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Window Manager Integrations'),
+      ),
+      body: Column(
+        children: <Widget>[
+          SizedBox(
+            height: 300,
+            child: AndroidView(
+              viewType: 'simple_view',
+              onPlatformViewCreated: onPlatformViewCreated,
+            ),
+          ),
+          if (lastTestStatus != _LastTestStatus.pending) _statusWidget(),
+          if (viewChannel != null) RaisedButton(
+            key: const ValueKey<String>('ShowAlertDialog'),
+            child: const Text('SHOW ALERT DIALOG'),
+            onPressed: onShowAlertDialogPressed,
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _statusWidget() {
+    assert(lastTestStatus != _LastTestStatus.pending);
+    final String message = lastTestStatus == _LastTestStatus.success ? 'Success' : lastError;
+    return Container(
+      color: lastTestStatus == _LastTestStatus.success ? Colors.green : Colors.red,
+      child: Text(
+        message,
+        key: const ValueKey<String>('Status'),
+        style: TextStyle(
+          color: lastTestStatus == _LastTestStatus.error ? Colors.yellow : null,
+        ),
+      ),
+    );
+  }
+
+  Future<void> onShowAlertDialogPressed() async {
+    if (lastTestStatus != _LastTestStatus.pending) {
+      setState(() {
+        lastTestStatus = _LastTestStatus.pending;
+      });
+    }
```

Comment:
Why did we need this step? It seems the status is going to become `success` or `error` in a moment later anyways?

Reply:
I see now it is to make sure the status is pending because we never set the status back to pending.  If we make change for my other comment to have a method to hide the alert, then we can set the status to pending in the `hideAlert` method.
We can also add a test for hide alert view.